packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -340,14 +401,24 @@ export default class Meeting extends StatelessSparkPlugin {
 
   /**
    *
-   * Update the audio options of this meeting
+   * toggle audio on or off (mute)
    * @param {object} [options]
+   * @returns {Promise}
+   * @memberof Meeting
+   */
+  toggleAudioVideo(options) {
```

Comment:
jsdoc doesn't match the method here. Are we going to have a single method to manipulate audio/video or individual `meeting.toggleAudio`. Is there value in making them more explicit `meeting.muteAudio` and `meeting.muteVideo`. I missed this part of the discussion in the design docs

Reply:
removed in new PR